src/solvers/nonlinear_solid_operators.hpp

```diff
@@ -42,44 +44,44 @@ class NonlinearSolidQuasiStaticOperator : public mfem::Operator {
 class NonlinearSolidReducedSystemOperator : public mfem::Operator {
  private:
   /// The bilinear form for the mass matrix
-  std::shared_ptr<mfem::ParBilinearForm> m_M_form;
+  std::shared_ptr<mfem::ParBilinearForm> M_form_;
 
   /// The bilinear form for the viscous terms
-  std::shared_ptr<mfem::ParBilinearForm> m_S_form;
+  std::shared_ptr<mfem::ParBilinearForm> S_form_;
 
   /// The nonlinear form for the hyperelastic response
-  std::shared_ptr<mfem::ParNonlinearForm> m_H_form;
+  std::shared_ptr<mfem::ParNonlinearForm> H_form_;
 
   /// The linearized jacobian
-  mutable std::unique_ptr<mfem::HypreParMatrix> m_jacobian;
+  mutable std::unique_ptr<mfem::HypreParMatrix> jacobian_;
 
   /// The current timestep
-  double m_dt;
+  double dt_;
 
   /// The current displacement and velocity vectors
-  const mfem::Vector *m_v, *m_x;
+  const mfem::Vector *v_, *x_;
 
   /// Working vectors
-  mutable mfem::Vector m_w, m_z;
+  mutable mfem::Vector w_, z_;
```

Comment:
I am warming up to this trailing underscore... so you win this round @samuelpmishLLNL (i think it was you who mentioned this).